R/pkg/R/utils.R

```diff
@@ -334,6 +334,27 @@ varargsToEnv <- function(...) {
   env
 }
 
+# Utility function to capture the varargs into environment object but all values are converted
+# into string.
+varargsToStrEnv <- function(...) {
+  pairs <- list(...)
+  env <- new.env()
+  for (name in names(pairs)) {
+    value <- pairs[[name]]
+    if (!(is.logical(value) || is.numeric(value) || is.character(value) || is.null(value))) {
+      stop("value[", value, "] in key[", name, "] is not convertable to string.")
```

Comment:
Oh, I see. I think I should the format and way to print out and error message.

How about something like..

```
"Supported types for options are logical, string, boolean, number and null.
The value set in ", name, " is ", typeof(value), "."
```